src/test/java/org/folio/fqm/utils/ItemStatusEnumTest.java

```diff
@@ -0,0 +1,65 @@
+package org.folio.fqm.utils;
+
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+class ItemStatusEnumTest {
+
+  @Test
+  void checkNumberOfItemStatuesDoNotChanged() {
+    try {
+
+      URL url = new URL("https://raw.githubusercontent.com/folio-org/mod-inventory-storage/master/ramls/item.json");
+
+      HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+      connection.setRequestMethod("GET");
+
+      int responseCode = connection.getResponseCode();
+
+      if (responseCode == HttpURLConnection.HTTP_OK) {
+        BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+        String inputLine;
+        StringBuilder content = new StringBuilder();
+
+        while ((inputLine = in.readLine()) != null) {
+          content.append(inputLine);
+        }
+        in.close();
```

Comment:
Tiny thing, but `BufferedReader` implements `AutoCloseable`, so it'd be a little better to do this in a try-with-resources block. It avoids the call to `close()` while also guaranteeing that `close()` definitely gets called, even if something throws an exception:

```suggestion
        try (BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()))) {
          String inputLine;
          StringBuilder content = new StringBuilder();

          while ((inputLine = in.readLine()) != null) {
            content.append(inputLine);
          }
        }
```

Reply:
Note: This puts `content` into the `try`-block's scope, so that would need to be fixed. This makes me really wish java supported something like
```
String content = () -> { ... do stuff; return something; };
```

...oh how I wish we could use Kotlin...